File: app/src/main/java/com/lmgy/exportapk/widget/RoundCheckBox.java
package com.lmgy.exportapk.widget;

import android.content.Context;
import android.util.AttributeSet;

import androidx.appcompat.widget.AppCompatCheckBox;

import com.lmgy.exportapk.R;

/**
 * @author lmgy
 * @date 2019/10/17
 */
public class RoundCheckBox extends AppCompatCheckBox {

    public RoundCheckBox(Context context) {
        this(context, null);
    }

    public RoundCheckBox(Context context, AttributeSet attrs) {
        this(context, attrs, R.attr.radioButtonStyle);
    }

    public RoundCheckBox(Context context, AttributeSet attrs, int defStyleAttr) {
        super(context, attrs, defStyleAttr);
    }


}

File: app/src/main/java/com/lmgy/exportapk/widget/RoundCheckBox.kt
package com.lmgy.exportapk.widget

import android.content.Context
import android.util.AttributeSet
import androidx.appcompat.widget.AppCompatCheckBox
import com.lmgy.exportapk.R

/**
 * @author lmgy
 * @date 2019/10/16
 */
class RoundCheckBox: AppCompatCheckBox {

    constructor(context: Context): this(context, null)

    constructor(context: Context, attrs: AttributeSet?): this(context, attrs, R.attr.radioButtonStyle)

    constructor(context: Context, attrs: AttributeSet?, defStyleAttr: Int): super(context, attrs, defStyleAttr)

}